src/IssueViz.Security.UnitTests/Taint/TaintList/TaintIssuesControlViewModelTests.cs

```diff
@@ -147,22 +141,17 @@ public void Ctor_ActiveDocumentExists_IssuesFilteredForActiveFilePath()
             var issueViz2 = CreateIssueViz("test2.cpp");
             var storeCollection = new ObservableCollection<IAnalysisIssueVisualization> { issueViz1, issueViz2 };
 
-            var activeDocument = new Mock<ITextDocument>();
-            activeDocument.Setup(x => x.FilePath).Returns("test2.cpp");
-
-            var activeDocumentLocator = new Mock<IActiveDocumentLocator>();
-            activeDocumentLocator.Setup(x => x.FindActiveDocument()).Returns(activeDocument.Object);
+            var locator = CreateLocatorAndSetActiveDocument("test2.cpp");
 
-            var testSubject = CreateTestSubject(storeCollection, activeDocumentLocator: activeDocumentLocator.Object);
+            var testSubject = CreateTestSubject(storeCollection, activeDocumentLocator: locator);
 
             testSubject.Issues.Count.Should().Be(2);
 
-            var view = CollectionViewSource.GetDefaultView(testSubject.Issues);
-            view.Filter.Should().NotBeNull();
+            VerifyFilterIsNotNull(testSubject);
 
-            var filteredItems = testSubject.Issues.Where(x => view.Filter(x));
```

Comment:
It's better to check the contents of the view, rather then the test manually applying the filter again.